.github/workflows/build-test.yml

```diff
@@ -201,4 +206,33 @@ jobs:
       - name: Build with Maven
         run: |
           mvn clean install -pl -:gremlin-javascript,-:gremlin-python,-gremlin-dotnet,-:gremlin-dotnet-source,-:gremlin-dotnet-tests -q -DskipTests -Dci
-          mvn verify -pl :neo4j-gremlin -DincludeNeo4j
\ No newline at end of file
+          mvn verify -pl :neo4j-gremlin -DincludeNeo4j
+  go:
+    name: Go
+    timeout-minutes: 20
+    needs: smoke
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go version
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go_version }}
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2.0.0
+
+      - name: Build
+        run: go build -v ./...
+
+      - name: Test and generate coverage
+        run: go test -v ./... -race -covermode=atomic -coverprofile="coverage.out"
```

Comment:
When run locally fails with runtime error:
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x14dd782]
```

Reply:
Is there a gremlin-server running locally?

Reply:
No, this is the issue if there is no gremlin server running locally, application/tests should fail gracefully with the meaningful error message, not with `invalid memory address or nil pointer dereference`. This is a negative case to improve usability.